dispatcher.php

```diff
@@ -39,14 +39,19 @@ public function __construct() {
 		//if requested url starts with api_base_url()
 		if ( false !== strpos( $_SERVER['REQUEST_URI'], get_api_base() ) ) {
 			require_once( 'api/v1/API.php' );
-
-			add_action( 'wp_loaded', array( $this, 'dispatch_api' ) );
+			
+			// Add API dispatch action
```

Comment:
Remove 		-			add_action( 'wp_loaded', array( $this, 'dispatch_api' ) );
Also remove line 42 		+			// Add API dispatch action